doc/api/brotli.md

```diff
@@ -0,0 +1,404 @@
+* `brotli.constants.BROTLI_DECODER_ERROR_FORMAT_CL_SPACE`
+* `brotli.constants.BROTLI_DECODER_ERROR_FORMAT_HUFFMAN_SPACE`
+* `brotli.constants.BROTLI_DECODER_ERROR_FORMAT_CONTEXT_MAP_REPEAT`
+* `brotli.constants.BROTLI_DECODER_ERROR_FORMAT_BLOCK_LENGTH_1`
+* `brotli.constants.BROTLI_DECODER_ERROR_FORMAT_BLOCK_LENGTH_2`
+* `brotli.constants.BROTLI_DECODER_ERROR_FORMAT_TRANSFORM`
+* `brotli.constants.BROTLI_DECODER_ERROR_FORMAT_DICTIONARY`
+* `brotli.constants.BROTLI_DECODER_ERROR_FORMAT_WINDOW_BITS`
+* `brotli.constants.BROTLI_DECODER_ERROR_FORMAT_PADDING_1`
+* `brotli.constants.BROTLI_DECODER_ERROR_FORMAT_PADDING_2`
+* `brotli.constants.BROTLI_DECODER_ERROR_FORMAT_DISTANCE`
+* `brotli.constants.BROTLI_DECODER_ERROR_DICTIONARY_NOT_SET`
+* `brotli.constants.BROTLI_DECODER_ERROR_INVALID_ARGUMENTS`
+* `brotli.constants.BROTLI_DECODER_ERROR_ALLOC_CONTEXT_MODES`
+* `brotli.constants.BROTLI_DECODER_ERROR_ALLOC_TREE_GROUPS`
+* `brotli.constants.BROTLI_DECODER_ERROR_ALLOC_CONTEXT_MAP`
+* `brotli.constants.BROTLI_DECODER_ERROR_ALLOC_RING_BUFFER_1`
+* `brotli.constants.BROTLI_DECODER_ERROR_ALLOC_RING_BUFFER_2`
+* `brotli.constants.BROTLI_DECODER_ERROR_ALLOC_BLOCK_TYPE_TREES`
+* `brotli.constants.BROTLI_DECODER_ERROR_UNREACHABLE`
+
+Chunk sizes.
+
+* `brotli.constants.BROTLI_MIN_CHUNK`
+* `brotli.constants.BROTLI_MAX_CHUNK`
+* `brotli.constants.BROTLI_DEFAULT_CHUNK`
+
+Compression mode.
+
+* `brotli.constants.BROTLI_MODE_GENERIC`
+* `brotli.constants.BROTLI_MODE_TEXT`
+* `brotli.constants.BROTLI_MODE_FONT`
+* `brotli.constants.BROTLI_DEFAULT_MODE`
+
+Quality levels.
+
+* `brotli.constants.BROTLI_MIN_QUALITY`
+* `brotli.constants.BROTLI_MAX_QUALITY`
+* `brotli.constants.BROTLI_DEFAULT_QUALITY`
+
+LGWIN values.
+
+* `brotli.constants.BROTLI_MIN_WINDOW_BITS`
+* `brotli.constants.BROTLI_MAX_WINDOW_BITS`
+* `brotli.constants.BROTLI_LARGE_MAX_WINDOW_BITS`
+* `brotli.constants.BROTLI_DEFAULT_WINDOW`
+
+LGBLOCK values.
+
+* `brotli.constants.BROTLI_MIN_INPUT_BLOCK_BITS`
+* `brotli.constants.BROTLI_MAX_INPUT_BLOCK_BITS`
+
+NPOSTFIX values.
+
+* `brotli.constants.BROTLI_MAX_NPOSTFIX`
+
+Parameters.
+
+* `brotli.constants.BROTLI_PARAM_MODE`
+* `brotli.constants.BROTLI_PARAM_QUALITY`
+* `brotli.constants.BROTLI_PARAM_LGWIN`
+* `brotli.constants.BROTLI_PARAM_LGBLOCK`
+* `brotli.constants.BROTLI_PARAM_DISABLE_LITERAL_CONTEXT_MODELING`
+* `brotli.constants.BROTLI_PARAM_SIZE_HINT`
+* `brotli.constants.BROTLI_PARAM_LARGE_WINDOW`
+* `brotli.constants.BROTLI_PARAM_NPOSTFIX`
+* `brotli.constants.BROTLI_PARAM_NDIRECT`
+* `brotli.constants.BROTLI_DECODER_PARAM_DISABLE_RING_BUFFER_REALLOCATION`
+* `brotli.constants.BROTLI_DECODER_PARAM_LARGE_WINDOW`
+
+## Class: Options
+<!--type=misc-->
+
+Each class takes an `options` object. All options are optional.
+
+Note that some options are only relevant when compressing, and are
+ignored by the decompression classes.
+
+* `chunkSize` {integer}
+**Default:** `16 * 1024`
```

Comment:
For better readability of the source, maybe it is worth to indent all the wrapped chunks:
```md
* `chunkSize` {integer}
  **Default:** `16 * 1024`
```
And maybe some short lines may be left unwrapped.